### ISSUE TITLE
Include additional instructions for configuring audio event?

### ISSUE THREAD

**Issue by bblackbourn** [MicrosoftDocs/mixed-reality#2118]
The style of these tutorials suggest that they are beginner level (eg lot's of reminders etc) but currently there's no explanation of what "configure the Octa object" entails. 

Currently reads:
Then configure the Octa object to receive the event and define AudioSource.PlayOneShot as the action to be triggered....

It feels like a step has gone missing, so it might be worth moving the tip about how to implement events up higher or offer alternative approach by clarifying that the user could do something along the lines of:  directly drag the icon/text of the OCTA (AUDIO SOURCE) component above onto the object field of the newly created "On Touch Started" Event so that the Octa (Audio Source) component object (rather than "the Octa object" in this case) receives the event.  Then, from the Function dropdown select  AudioSource&gt;AudioSource.PlayOneShot as the action to be triggered....

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a7e480f6-30d8-9538-bff1-74ba54dc310e
* Version Independent ID: 1cb31b72-b828-d8d3-74fa-6641a61f3750
* Content: [Getting started tutorials - 5. Interacting with 3D objects - Mixed Reality](https://docs.microsoft.com/en-gb/windows/mixed-reality/mrlearning-base-ch4)
* Content Source: [mixed-reality-docs/mrlearning-base-ch4.md](https://github.com/MicrosoftDocs/mixed-reality/blob/master/mixed-reality-docs/mrlearning-base-ch4.md)
* Product: **mixed-reality**
* Technology: **mr-dev**
* GitHub Login: @jessemcculloch
* Microsoft Alias: **jemccull**

**Comment by martinwilter** [comment]
@bblackbourn Thanks for your suggestions, we really appreciate it.

Yes, these tutorials are meant to be beginner level. However, we try to keep the repetition of step-by-step instructions to a minimum so the reader gets the benefit of learning by recollection. 

Because events were covered extensively in the [Creating user interface and configure Mixed Reality Toolkit](https://docs.microsoft.com/en-gb/windows/mixed-reality/mrlearning-base-ch2) tutorial, we decided not to provide step-by-step again here, but instead provide the Tip annotation with link to the instructions covering it in detail.
 
The instructions for this section, [4. Implement the On Touch Started event](https://docs.microsoft.com/en-gb/windows/mixed-reality/mrlearning-base-ch4#4-implement-the-on-touch-started-event), contains two sub-steps, so we placed the Tip at the end of that section, allowing readers to try to recollect how to do it before any tips are given. Additionally, where possible, we try to keep the annotations to the start or end of the sections to minimize reading interruptions.

If you still feel the Tip needs to be moved up, let us know, and we will reconsider.
